rest/route/select_test.go

```diff
@@ -0,0 +1,107 @@
+package route
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSelectTestsHandler(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	j := []byte(`{
+		"project": "my-project",
+		"requester": "patch",
+		"build_variant": "variant",
+		"task_id": "my-task-1234",
+		"task_name": "my-task",
+		"tests": ["test1", "test2", "test3"]
+	}`)
+	req, _ := http.NewRequest(http.MethodPost, "/select/tests", bytes.NewBuffer(j))
+	sth := makeSelectTestsHandler()
+	assert.NoError(t, sth.Parse(ctx, req))
```

Comment:
nit: assert -> require

Reply:
fixed

Reply:
No need to change this back Brian but for future reference @ZackarySantana , we usually only use require when continuing would cause a panic or something (such as checking length on a list that we're about to go into); otherwise, its helpful for testing to see all errors at once rather than fixing one at a time.

Reply:
FWIW this is something about which reasonable people disagree. An advantage for `require` is that you can quickly end a long-running test, so it can make iterating faster.